jupyter-book/conditions/perturbation_modeling.ipynb

```diff
@@ -22,12 +22,12 @@
    "metadata": {},
```

Comment:
<p>The explanation of the dataset here is abysmal. Please just say "Bhattacherjee et al. offered mice cocaine, and took samples of the prefrontal cortex for scRNA-seq at 15hrs and 48hrs post withdrawal." I'm assuming there is also a "no cocaine" condition not mentioned.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/cross-modal-single-cell-best-practices/pull/91/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>